include/net/fota_download.h

```diff
@@ -145,13 +138,38 @@ int fota_download_init(fota_download_callback_t client_callback);
  * @retval 0	     If download has started successfully.
  * @retval -EALREADY If download is already ongoing.
  * @retval -E2BIG    If sec_tag_count is larger than
- *		     @kconfig{CONFIG_FOTA_DOWNLOAD_SEC_TAG_LIST_SIZE_MAX}
+ *		     `CONFIG_FOTA_DOWNLOAD_SEC_TAG_LIST_SIZE_MAX`
```

Comment:
why the change? The old way should create a direct link in the generated documentation which goes to the Kconfig page, why would you want to remove that?

Reply:
according to @peknis these are not supported anymore in the docs

Reply:
Keeping the macro has some advantages:

- When reading code, it's clear this is a Kconfig
- Allows us to tweak it in the future to, maybe, link somewhere
- We can also tweak the style

So please, keep using `@kconfig` even if it just renders a verbatim block. Upstream Zephyr also does this, so let's be consistent.

Reply:
Is it okay if I revert to the old way and we move this discussion to a new PR? There are many other places that still use the old macro. @peknis

Reply:
I'd still argue it renders ugly and cuts the text flow. I would rather have it just in single backticks. But it has its advantages when reading the code. Whatever.

Reply:
See, for example https://docs.nordicsemi.com/bundle/nrf-apis-2.9.0/page/group_lte_lc_ga4c137ae6b3b1c7006dd730458958fb77.html#ga4c137ae6b3b1c7006dd730458958fb77

It does not look very good.

Reply:
that's because of that hideous zoom thing though, it looks absolutely fine without it https://ncsdoc.z6.web.core.windows.net/latest/nrf/doxygen/html/group__lte__lc_ga4c137ae6b3b1c7006dd730458958fb77.html#ga4c137ae6b3b1c7006dd730458958fb77

Reply:
I'll open a PR just for that change.

Reply:
That sounds like an argument that we can't adopt everything from upstream and need to make adaptations in NCS

Reply:
@peknis @nordicjm @gmarull Please continue the discussion here: https://github.com/nrfconnect/sdk-nrf/pull/20022